controllers/imageclusterinstall_controller.go

```diff
@@ -574,6 +578,89 @@ func (r *ImageClusterInstallReconciler) removeBMHImage(ctx context.Context, bmhR
 	return nil
 }
 
+func setBackupLabel(obj client.Object) bool {
+	labels := obj.GetLabels()
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+	if labels[backupLabel] == backupLabelValue {
+		return false
+	}
+
+	labels[backupLabel] = backupLabelValue
+	obj.SetLabels(labels)
+	return true
+}
+
+func (r *ImageClusterInstallReconciler) labelConfigMapForBackup(ctx context.Context, key types.NamespacedName) error {
+	cm := &corev1.ConfigMap{}
+	if err := r.Get(ctx, key, cm); err != nil {
+		return err
+	}
+
```

Comment:
Consider checking if the label already exists and return

Reply:
This is being done based on the return value of `setBackupLabel`.

I did it this way to share the logic between secrets and config maps.